package.json

```diff
@@ -98,6 +101,7 @@
     "bson": "^4.0.3",
     "chi-squared": "^1.1.0",
     "codepage": "^1.14.0",
+    "commander": "^2.14.1",
```

Comment:
I chose this version of commander because it's also the version pulled by codepage, in the hopes of minimizing impact of adding a dependency.